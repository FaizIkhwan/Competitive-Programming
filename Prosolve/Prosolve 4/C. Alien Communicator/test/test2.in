3
hey hooman
we heard that yall are idiot
is that true?