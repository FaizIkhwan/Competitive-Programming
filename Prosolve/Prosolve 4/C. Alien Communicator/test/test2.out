hooman hey
idiot are yall that heard we
true? that is
